app/views/teams/_show_mobile.html.erb

```diff
@@ -54,12 +53,12 @@
         <% end %>
       <% end %>
     </div>
-    <div id="user" data-teams-tab-change-target="teamContent" class="" data-tab="user">
+    <div id="user" data-teams-tab-change-target="teamContent" class="pt-6" data-tab="user" data-controller="user-show">
```

Comment:
Do we need user-show controller ?

Reply:
Can use use target name here to relate this link to the actual tab ?

Reply:
There are so many identifiers here id, target and data-tab, why do we need all these ?

Reply:
user-show controller removed as it is not using.also IDs are removed ,and targets used to get tab link